content/en/apps/payments/payments-api.md

```diff
@@ -0,0 +1,600 @@
+---
+date: '2020-01-08T09:59:25Z'
+menu:
+  apps:
+    parent: "payments"
+    name: Payment Service
+title: Payment Service
+weight: 400
+---
+Use Payment Service flows to initiate payments and account management requests from a node on a Corda network. These requests can then be picked up by the [Payments Agent](payments-agent.md) on your network.
+
+Use the Payment Service flows to:
+
+* Create a request for a payment to be made. [MakePayment](#makepayment).
+* Reinstate a stalled payment into the payment workflow. [ReinstatePayment](#reinstatepayment).
+* Cancel a stalled payment. [CancelPayment](#cancelpayment).
+* Update and reinstate a stalled payment into the payment workflow. [RemediatePayment](#remediatepayment).
+* Return the list of payments stalled on an error. [StalledPayments](#stalledpayments).
+* Return the payment state associated with a specific payment ID or provided transaction reference. [PaymentById](#paymentbyid).
+* Return the PSPs that are configured on the system and available to use. [AvailablePSPs](#availablepsps).
+* Return a PSP specification from the Payments Agent. [PspSpecification](#pspspecification).
+* Add a payment account for an existing account on the PSP. [AddPaymentAccount](#addpaymentaccount).
```

Comment:
Might want to change this as it took me a few reads: 
```suggestion
* Add a payment account for an existing PSP account. [AddPaymentAccount](#addpaymentaccount).
```
I'm reading the original text that whatever you're doing, you're doing it to the PSP, but I don't think that's right.